### DISCUSSION TITLE
FF Firmware Flasher not connecting?

### DISCUSSION THREAD

**Discussion by cdsmakestuff** [moonglow/FlashForge_Marlin#69] [General]
Anyone have an idea why I can't get FF Firmware Flasher to connect to my dreamer NX?

Can connect to it via FlashPrint... Drivers are working...

Just the flasher doesn't see machine during boot.

Would really like to try this firmware.

Thanks

**Comment by cdsmakestuff** [comment]
Got it to work!

Had to load the marlin firmware through flashprint.

Just rename the new firmware to the same as the existing firmware in the "flashprint/firmware" folder.

Thanks for work on this!

**Comment by moonglow** [comment]
@cdsmakestuff  Hi ! Try to put your firmware to the same folder with flasher and make sure what path to flasher does not has spaces or non english letters...  I think some of this must helps. If not let me know.
